src/main/java/org/bigredbands/mb/controllers/PDFGenerator.java

```diff
@@ -123,17 +125,23 @@ public void createPDF(DrillInfo drillInfo, File file) throws IOException {
                     pageHeight - bufferTop);
             // we have a map of measure number to count per measure
             // currently assumes 4 counts per measure
-            // TODO: Make measures adjust with changes in count per measure
-            begMeasure = endMeasure;
-            endMeasure = begMeasure + move.getCounts() / 4;
-            if (begMeasure != endMeasure) {
-                contentStream.showText("Measures:  " + (begMeasure + 1)
-                        + " - " + endMeasure);
-            } else {
-                contentStream.showText("Measures:  " + begMeasure + " - "
-                        + endMeasure);
+            int totalCounts = extraCounts;
+            while (move.getCounts() > totalCounts){
+                if (drillInfo.getCountsHashMap().get(currentMeasure) != null) {
+                    currentCountsPerMeasure = drillInfo.getCountsHashMap().get(currentMeasure);
+                }
```

Comment:
I tested this with an drill file from a past year's show (where we inserted the drum feature as one _long_ measure), and noticed there was an off-by-one error in the measure numbers in the PDF. This change fixes it.

```suggestion
                currentCountsPerMeasure = drillInfo.getCountsHashMap().getOrDefault(currentMeasure + 1, currentCountsPerMeasure);
```

Reply:
Sorry for not communicating this earlier but I went on vacation out of the country for 2 weeks. I'm back now and I'll get this change done asap. I hope you're doing well!